Doc/library/stdtypes.rst

```diff
@@ -2136,7 +2136,11 @@ expression support in the :mod:`re` module).
         >>> "they're bill's friends from the UK".title()
         "They'Re Bill'S Friends From The Uk"
 
-   A workaround for apostrophes can be constructed using regular expressions::
+   The :func:`string.capwords` function does not have this problem, as it
+   splits words on spaces first.
```

Comment:
```suggestion
   splits words on spaces only.
```
This seems slightly clearer. What capwords fixes is that it doesn't split on apostrophes, not the order in which it does things.